### PR TITLE
feat: run OCR extraction on every new image

### LINKED ISSUE #320: Proof: send to Google Cloud Vision, and store the matching JSON output
### Problem

- we should probably send all prices receipts and prices images to Google Cloud Vision, and store the matching JSON output

### Links

- https://cloud.google.com/vision/docs
- https://github.com/openfoodfacts/robotoff/blob/ef962195e66236a401356573595b0861186394e0/scripts/ocr/run_ocr.py#L32
- https://github.com/openfoodfacts/openfoodfacts-python/blob/develop/openfoodfacts/ocr.py#L1201 & https://github.com/openfoodfacts/openfoodfacts-python/blob/develop/openfoodfacts/ocr.py#L295
- https://github.com/clovaai/cord

### Why

- NSFW detection
- Siret Detection/Basic receipt data for simplistic pre-population
- Written barcode on shelf detection
- …
- detect the number of items, and make it available so that we can nudge the user properly
- detect the total
- detect date
- detect location (country at least)
- detect currency

### PR DESCRIPTION
**Pull request by raphael0202**
Fixes #320

### CODE REVIEW

File: open_prices/proofs/utils.py [hunk 1]
```diff
@@ -124,3 +133,73 @@ def store_file(
...
+
+    data = run_ocr_on_image(image_path, api_key)
+
+    if data is None:
+        return
+
+    data["created_at"] = int(time.time())
+
+    with gzip.open(ocr_json_path, "wt") as f:
+        f.write(json.dumps(data))
```
Comment (raphodn) [inline open_prices/proofs/utils.py:203]: so it stores the result in a jsonl.gz file next to the image ?

Comment (raphodn) [reply open_prices/proofs/utils.py:203]: if we re-run on the same image, will it override ?

Comment (raphael0202) [reply open_prices/proofs/utils.py:203]: > so it stores the result in a jsonl.gz file next to the image ?
Yes exactly!

Comment (raphael0202) [reply open_prices/proofs/utils.py:203]: > if we re-run on the same image, will it override ?

It depends on the value of `override`

Comment (raphodn) [reply open_prices/proofs/utils.py:203]: ah ouiii bien vu !

**Comment by raphodn** [comment]
todo : avoid running OCR extraction in the testruns + delete images

File: open_prices/api/proofs/views.py [hunk 2]
```diff
@@ -75,6 +77,10 @@ def upload(self, request: Request) -> Response:
                 status=status.HTTP_400_BAD_REQUEST,
             )
         file_path, mimetype, image_thumb_path = store_file(request.data.get("file"))
+        async_task(
```
[the remark this answers is not in the document]
Comment (raphodn) [reply open_prices/api/proofs/views.py:80]: done in #549

**Pull request merged**